WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -226,60 +227,83 @@ class MySiteViewModel @Inject constructor(
         backupAvailable: Boolean,
         scanAvailable: Boolean,
         cards: CardsResult<List<CardModel>>?
-    ) = cardsBuilder.build(
-            DomainRegistrationCardBuilderParams(
-                    isDomainCreditAvailable = isDomainCreditAvailable,
-                    domainRegistrationClick = this::domainRegistrationClick
-            ),
-            PostCardBuilderParams(
-                    posts = cards?.model?.firstOrNull { it is PostsCardModel } as? PostsCardModel,
-                    onPostItemClick = this::onPostItemClick,
-                    onFooterLinkClick = this::onPostCardFooterLinkClick
-            ),
-            QuickActionsCardBuilderParams(
-                    siteModel = site,
-                    activeTask = activeTask,
-                    onQuickActionStatsClick = this::quickActionStatsClick,
-                    onQuickActionPagesClick = this::quickActionPagesClick,
-                    onQuickActionPostsClick = this::quickActionPostsClick,
-                    onQuickActionMediaClick = this::quickActionMediaClick
-            ),
-            QuickStartCardBuilderParams(
-                    quickStartCategories = quickStartCategories,
-                    onQuickStartBlockRemoveMenuItemClick = this::onQuickStartBlockRemoveMenuItemClick,
-                    onQuickStartTaskTypeItemClick = this::onQuickStartTaskTypeItemClick
-            ),
-            SiteInfoCardBuilderParams(
-                    site = site,
-                    showSiteIconProgressBar = showSiteIconProgressBar,
-                    titleClick = this::titleClick,
-                    iconClick = this::iconClick,
-                    urlClick = this::urlClick,
-                    switchSiteClick = this::switchSiteClick,
-                    activeTask = activeTask
-            )
-    ) + dynamicCardsBuilder.build(
-            quickStartCategories,
-            pinnedDynamicCard,
-            visibleDynamicCards,
-            this::onDynamicCardMoreClick,
-            this::onQuickStartTaskCardClick
-    ) + siteItemsBuilder.build(
-            SiteItemsBuilderParams(
-                    site = site,
-                    activeTask = activeTask,
-                    backupAvailable = backupAvailable,
-                    scanAvailable = scanAvailable,
-                    onClick = this::onItemClick
-            )
-    )
+    ): List<MySiteCardAndItem> {
+        val cards = cardsBuilder.build(
```

Comment:
Nitpick: There's a name shadow AS warning here. We might want to change the `cards` param to `cardsResult`.